README.rst

```diff
@@ -8,15 +8,31 @@ python-crfsuite
 .. image:: https://ci.appveyor.com/api/projects/status/uaq4sw8tc0oojr4v?svg=true
     :target: https://ci.appveyor.com/project/kmike/python-crfsuite-mhi5h
 
+.. image:: https://img.shields.io/github/release/scrapinghub/python-crfsuite.svg?style=flat-square
+    :target: https://github.com/scrapinghub/python-crfsuite/releases
+    :alt: Current Release Version
```

Comment:
This badge shows "none" as a release version, is this intended?
I don't want to use github Releases feature; if that's automatic and badge will be updated then it is fine to keep it, but if it requires more work on releases, duplicating changelog, etc., then let's drop this badge.